moped-database/migrations/1699899884388_atsd_component_tags/up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE moped_component_tags DROP CONSTRAINT moped_component_tags_name_key;
+ALTER TABLE public.moped_component_tags add constraint moped_component_tags_name_key UNIQUE ("name", "type");
+
+INSERT INTO public.moped_component_tags (name, "type", slug)
+        values
+        ( '2024', 'MAP Ped Crossing', 'map_ped_crossing_2024'),
+        ('All', 'Ped Crossing Priority', 'ped_crossing_priority_all'),
+        ('2024', 'Bikeways – Work Plan', 'bikeways_work_plan_2025');
```

Comment:
I think that this slug needs to be updated to say 2024.

Also, tiny detail but I noticed that the existing bikeways work plan tags use a short dash instead of the long dash that is added here. I caught it because it is more noticeable in the UI.
**Bikeways - Work Plan**
**Bikeways – Work Plan**

Reply:
wow—great catch on both counts. thank you!!!!